src/components/Info/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { faArrowAltCircleRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  Container,
+} from 'react-bootstrap';
+
+function Info() {
```

Comment:
Take look at the image below.
- [ ] In the Info component, we are missing some parts of the header design as we have in the template. Let's add this.
- [ ] Also we are missing the **striped color pattern** for the country details. Kindly add this.

![adewu-detail](https://user-images.githubusercontent.com/36542986/142659501-ac86dde3-173b-494f-a32a-b5d8909593e6.jpg)